scripts/startup/set_panoptes_env_vars.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Why not the same as `scripts/install/default-env-vars.sh` (which has your `-z` additions)?

Reply:
Because I'm an idiot? I should have suggested that you document the file as intended to be modified to suit local conditions and then copied into /etc/profile.d/.

As it is, `default-env-vars.sh` is well suited to running from $HOME/.profile at a point AFTER the user has provided any non-standard values.